server.js

```diff
@@ -0,0 +1,96 @@
+//Set up
```

Comment:
Nice work with this server! If you want some extra reps before the Sprint Challenge this afternoon, I'd recommend filling in the gaps so you have a full set of CRUD operations for posts, users, and tags.